package.json

```diff
@@ -20,21 +20,21 @@
   "author": "Jordan Foreman <me@jordanforeman.com>",
   "license": "MIT",
   "dependencies": {
-    "aws-sdk": "^2.1126.0",
-    "yargs": "^15.0.0"
+    "aws-sdk": "^2.1469.0",
```

Comment:
:x: this actually breaks `silent` mode, as the `aws-sdk` notification doesn't respect the flag as its coded in the system. We'll need to either [suppress this deprecation](https://stackoverflow.com/questions/75479425/suppress-aws-sdk-v2-deprecation-message) notification or hold off on this update.

Reply:
> ❌ this actually breaks `silent` mode, as the `aws-sdk` notification doesn't respect the flag as its coded in the system. We'll need to either [suppress this deprecation](https://stackoverflow.com/questions/75479425/suppress-aws-sdk-v2-deprecation-message) notification or hold off on this update.

So we better return to the previous version, okay?

Reply:
Yes! Thanks for the quick work on this.